crates/ef-testing/src/evm_sequencer/evm_state/v0.rs

```diff
@@ -38,15 +43,43 @@ impl Evm for KakarotSequencer {
         let high_fee = base_fee >> U256::from(128);
         let high_fee: u128 = high_fee.try_into().unwrap(); // safe unwrap <= U128::MAX.
 
-        let base_address = get_storage_var_address("base_fee", &[]);
+        let basefee_address = get_storage_var_address("base_fee", &[]);
         self.state_mut()
-            .set_storage_at(kakarot_address, base_address, StarkFelt::from(low_fee));
+            .set_storage_at(kakarot_address, basefee_address, StarkFelt::from(low_fee));
         self.state_mut().set_storage_at(
             kakarot_address,
-            next_storage_key(&base_address)?,
+            next_storage_key(&basefee_address)?,
             StarkFelt::from(high_fee),
         );
 
+        // Set the previous randao.
+        let prev_randao_low = prev_randao & U256::from(u128::MAX);
```

Comment:
There should be a helper function split_u256 in the ef tests I think (not on pc so can't check)